cleanlab/datalab/internal/issue_manager/outlier.py

```diff
@@ -94,7 +95,7 @@ def find_issues(
         pred_probs: Optional[np.ndarray] = None,
         **kwargs,
     ) -> None:
-        knn_graph = self._process_knn_graph_from_inputs(kwargs)
+        knn_graph = ConstructedKNNGraph(self.datalab).process_knn_graph_from_inputs(kwargs)
```

Comment:
For the changed issue managers, consider creating an internal attribute in `__init__` for the `ConstructedKNNGraph` object. Something like 

`self._constructed_knn_graph_util = ConstructedKNNGraph(self.datalab)`

Then use the object to call `self._constructed_knn_graph_util.process_knn_graph_from_inputs(kwargs)`. 

This way, if more methods are added to the `ConstructedKNNGraph` class, you need not instantiate the object again to use those methods.